src/commands/owner-commands/debugerror.ts

```
@@ -1,24 +1,24 @@
+      throw new Error("Testing the error checking - debug only");
+    } catch (error) {
+      await logError("Debug error thrown:", error);
+      await interaction.createMessage({
+        content: "An error occurred: " + error.message,
+        flags: 64, // Ephemeral flag
+      });
     }
-}
\ No newline at end of file
+  }
+}
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)